build/testBuildImages.sh

```diff
@@ -23,6 +23,15 @@ else
     $buildBuildImagesScript "$@"
 fi
 
+testResult=$(dotnet test --filter "$MISSING_CATEGORY_FILTER" "$TESTS_SRC_DIR/$INTEGRATION_TEST_PROJECT/$INTEGRATION_TEST_PROJECT.csproj")
```

Comment:
these tests can fail or can be successful. I think we should let It run instead of forcing them to fail